Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -467,8 +467,8 @@ public bool WeaponEnvDamage(DaggerfallUnityItem strikingWeapon, RaycastHit hit)
                 return true;
             }
 
-            // Check if player hit a static exterior door
-            if (GameManager.Instance.PlayerActivate.AttemptExteriorDoorBash(hit))
+            // Check if player hit a static door
+            if (GameManager.Instance.PlayerActivate.AttemptStaticDoorBash(hit))
```

Comment:
I think you forgot to revert this change too?

Reply:
Hmm, looks like something went wrong when I was merging. It's fixed now.